compact.go

```diff
@@ -0,0 +1,113 @@
+package bbolt
+
+// Compact will create a copy of the source DB and in the destination DB. This may
+// reclaim space that the source database no longer has use for. txMaxSize can be
+// used to limit the transactions size of this process and may trigger intermittent
+// commits. A value of zero will ignore transaction sizes.
+func Compact(dst, src *DB, txMaxSize int64) error {
```

Comment:
Optional: Please consider adding  TODO: to merge with: 
https://github.com/etcd-io/etcd/blob/b7f0f52a16dbf83f18ca1d803f7892d750366a94/mvcc/backend/backend.go#L349

Reply:
Done.